### DISCUSSION TITLE
HG08673

### DISCUSSION THREAD

**Discussion by Smurphy1965** [Koenkk/zigbee2mqtt#19004] [Devices]
When i pair a HG08673 smartswitch into HA, i can see all settings except the on/off switch.
In the control tab there is only the child lock and indicator mode.
Did fw update, deletend and pair again, but all with same result.
Please advice what to do. 
Current version: 1.33.0-1

**Comment by Tabisch** [comment]
Have a bunch of those in HA 2023.9.2 and they work as expected.
Which version are you running.
Maybe its something about HA thats broken.

In Z2M i have legacy entity attributes and legacy triggers disabled.

**Comment by Smurphy1965** [comment]
HA is 2023.6.0.
It all worked before with other HG08673's

**Comment by francisp2** [comment]
Z2M 1.33 needs HA 2023.8 or higher. It is in the 1.33 release notes.
